tests/test_llms_text-generation_vllm-ray.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    ## Build VLLM Ray docker
+    cd $WORKPATH/comps/llms/text-generation/vllm-ray/docker
+    docker build \
+        -f Dockerfile.vllmray  \
+        -t vllm_ray:habana --network=host ../../
+
+    ## Build OPEA microservice docker
+    cd $WORKPATH
+    docker build \
+        -t opea/llm-vllm-ray:latest \
+        -f comps/llms/text-generation/vllm-ray/docker/Dockerfile.microservice .
+}
+
+function start_service() {
+    export LLM_MODEL="facebook/opt-125m"
+    port_number=8006
+    docker run -d --rm \
+        --name="vllm-ray-service" \
```

Comment:
`--name=xxx` -> `--name=test-comps-xxx`  
We use this prefix for GenAIComps test containers, to make it separate with GenAIExamples CI test. We have limit HW resource, so we should make sure the name won't be conflict.

Reply:
updated